src/phoenix/experimental/evals/models/openai.py

```diff
@@ -27,9 +27,9 @@ class OpenAiModel(BaseEvalModel):
     openai_api_key: Optional[str] = field(repr=False, default=None)
     openai_api_base: Optional[str] = field(repr=False, default=None)
     openai_organization: Optional[str] = field(repr=False, default=None)
-    model_name: str = "text-davinci-003"
+    model_name: str = "gpt-4"
```

Comment:
intentional?

Reply:
Yes, I figure since the main purpose of these models is currently to run evaluations, it's better to make the default settings reasonable choices for that task.